rustup-init.sh

```diff
@@ -224,85 +226,84 @@ get_architecture() {
     esac
 
     case "$_cputype" in
-
         i386 | i486 | i686 | i786 | x86)
-            local _cputype=i686
+            _cputype=i686
             ;;
 
         xscale | arm)
-            local _cputype=arm
+            _cputype=arm
             if [ "$_ostype" = "linux-android" ]; then
-                local _ostype=linux-androideabi
+                _ostype=linux-androideabi
             fi
             ;;
 
         armv6l)
-            local _cputype=arm
+            _cputype=arm
             if [ "$_ostype" = "linux-android" ]; then
-                local _ostype=linux-androideabi
+                _ostype=linux-androideabi
             else
-                local _ostype="${_ostype}eabihf"
+                _ostype="${_ostype}eabihf"
             fi
             ;;
 
         armv7l | armv8l)
-            local _cputype=armv7
+            _cputype=armv7
             if [ "$_ostype" = "linux-android" ]; then
-                local _ostype=linux-androideabi
+                _ostype=linux-androideabi
             else
-                local _ostype="${_ostype}eabihf"
+                _ostype="${_ostype}eabihf"
             fi
             ;;
 
         aarch64)
-            local _cputype=aarch64
+            _cputype=aarch64
             ;;
 
         x86_64 | x86-64 | x64 | amd64)
-            local _cputype=x86_64
+            _cputype=x86_64
             ;;
 
         mips)
-            local _cputype="$(get_endianness $_cputype "" 'el')"
+            _cputype="$(get_endianness $_cputype "" 'el')"
             ;;
 
         mips64)
-            local _bitness="$(get_bitness)"
-            if [ $_bitness = "32" ]; then
-                if [ $_ostype = "unknown-linux-gnu" ]; then
+            _bitness="$(get_bitness)"
```

Comment:
This converts `_bitness` to a global.

Reply:
As I explained above, this doesn't covert `_bitness` to a global.

Reply:
`_bitness` is never set to local anywhere.